docs/agents/sematext-agent/ansible.md

```diff
@@ -0,0 +1,129 @@
+- ansible_host is the IP address of each host
+- ansible_port is the SSH host port
+- ansible_ssh_user is the SSH username
+- ansible_ssh_private_key_file is the private SSH certificate file of each host
+
+If your inventory file is named `hosts` then you can verify whether you can ping all your hosts with the following command:
+
+```ansible -i hosts all -m ping```
+
+If the hosts are reachable, you should expect a response like the one below:
+
+```
+server1 | SUCCESS => {
+    "ansible_facts": {
+        "discovered_interpreter_python": "/usr/bin/python3"
+    },
+    "changed": false,
+    "ping": "pong"
+}
+server2 | SUCCESS => {
+    "ansible_facts": {
+        "discovered_interpreter_python": "/usr/bin/python3"
+    },
+    "changed": false,
+    "ping": "pong"
+}
+```
+
+
+## Sematext Agent Ansible Role
+The Sematext Agent role contains everything required to install Sematext Agent in a specified list of hosts according to the Ansible playbook that will be used. It can be installed either through Ansible Galaxy or manually from the [ansible-install-sematext repository](https://github.com/sematext/ansible-install-sematext).
+
+
+### Install from Ansible Galaxy
+Install the [Sematext Agent role from Ansible Galaxy](https://galaxy.ansible.com/ui/standalone/roles/sematext/spm-monitor-install/) on your control node by issuying the following command:
+
+```ansible-galaxy role install sematext.spm-monitor-install```
+
+
+### Manual Installation
+If you want to manually install the role, clone the [ansible-install-sematext](https://github.com/sematext/ansible-install-sematext) repo in the control node and copy this repo to `~/.ansible/roles/sematext.spm-monitor-install`.
+
+
+### Supported Platforms
+Linux distributions:
+
+- Ubuntu
+    - xenial
+    - bionic
+    - focal
+    - jammy
+- Debian
+    - wheezy
+    - jessie
+    - stretch
+    - buster
+    - bullseye
+    - bookworm
+- Red Hat Enterprise Linux 6, 7, 8, 9
+- Fedora 38
+- Amazon Linux 2
+
+## Deploying Sematext Agent
+
+### Create an Infra App in Sematext Cloud
+After you get logged into Sematext Cloud at https://apps.sematext.com (or https://apps.eu.sematext.com if using Sematext Cloud Europe), the first step is to [create an Infra App](https://sematext.com/docs/monitoring/infrastructure/#create-an-infra-app) to start monitor your infrastructure. After entering the App name, select Ansible from the list of the environments. Make sure to copy the provided infra token, it will be needed later to pass it as a var in your Ansible playbook.
+
+### Install the agent using an Ansible playbook
+You can integrate the Sematext Ansible role in your existing playbooks or create a new one. Here is an example playbook:
+
+```
+- hosts: all
+  gather_facts: yes
+  become: yes
+  vars:
+    infra_token: INFRA_TOKEN
+  roles:
+    - { role: sematext.spm-monitor-install }
+
+```
+
+`INFRA_TOKEN` is your infra token that you copied earlier from Sematext Cloud.
+
```

Comment:
It is also possible to define [custom tags](https://sematext.com/docs/tags/custom-tags/) in your playbook, by adding the following tasks defining the tags:
```
  tasks:
    - name: Append tags to st-agent.yml
      blockinfile:
        path: /opt/spm/properties/st-agent.yml
        block: |
          tags: "env:dev, project:projectName, role:webfrontend"

    - name: Restart sematext-agent service
      service:
        name: sematext-agent
        state: restarted
```

Reply:
Updated. @michaelhigham make sure to add a reference in https://sematext.com/docs/tags/custom-tags/